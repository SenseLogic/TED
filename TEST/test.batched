SetInputFolder IN/
SetOutputFolder OUT/

ReadFiles /*.hpp

SetTabulationSpaceCount 4
ReplaceTabulations 0 $
RemoveLastSpaces 0 $
RemoveLastEmptyLines 0 $
AddEmptyLines $ 1

IgnoreFirstSpaces
IgnoreLastSpaces
FindLines% 0 $
    `#ifndef __\Uib_HPP__`
    `#define __\Uib_HPP__`
FindLines~% 0 $
    `#ifndef __\Uib__`
    `#define __\Uib__`
SetLines^ [ ]
    #pragma once
RemoveFirstSpaces^ 0 $ 4
FindLines^% $-2 1
    #endif
RemoveLines^ [ ]

PrintLines

CreateFolders
WriteFiles
