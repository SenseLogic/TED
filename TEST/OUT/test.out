replaced_text other_replaced_text "replaced_text\nreplaced_text\"replaced_text"
ReplacedIdentifier otherIdentifier "ReplacedIdentifier\nIdentifier\"otherIdentifier"
ReplacedUnquotedText otherReplacedUnquotedText "UnquotedText\"UnquotedText\"otherUnquotedText"
ReplacedUnquotedidentifier otherUnquoteddentifier "Unquotedidentifier\nUnquotedidentifier\"otherUnquotedidentifier"
QuotedText otherQuotedText "ReplacedQuotedText\nReplacedQuotedText\"otherReplacedQuotedText"
QuotedIDENTIFIER otherQuotedIDENTIFIER "ReplacedQuotedIDENTIFIER\nReplacedQuotedIDENTIFIER\"otherQuotedIDENTIFIER"
 	TAB INSERTED HERE !
